tests/common/prometheus-operator.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	"testing"
+)
+
+func TestPrometheusOperator(t *testing.T) {
+	t.Parallel()
+	kubeconfig := TestModuleGetStringGlobal(t, KubeConfigPath)
+
+	// LEAD HARBOR
+	testDashboard := TestModule{
```

Comment:
Should update command and test module instance name to reflect test

Reply:
Done